src/main/java/uk/gov/beis/els/api/categories/refrigeratordirectsales/BeverageCoolersInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.refrigeratordirectsales;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotBlank;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.RescaledInternetLabelApiForm;
+import uk.gov.beis.els.categories.refrigeratorsdirectsales.service.RefrigeratorsDirectSalesService;
+
+@Schema(name = "Beverage coolers arrow image")
+public class BeverageCoolersInternetLabelApiForm extends RescaledInternetLabelApiForm {
+
+  @Schema(description = "Energy efficiency class indicator")
```

Comment:
Should these say something more along the lines of "Energy efficiency class for beverage coolers"? And so on with the other ones.

Reply:
addressed